docs/pages/use-teleport/teleport-connect.mdx

```diff
@@ -0,0 +1,108 @@
+- The cluster selector (to the left of the command bar) shows up only if there are leaf clusters
+  connected to the root cluster. It lets you browse leaf cluster resources. It also changes
+  which cluster the commands in the command bar are going to target.
+
+## Connecting to an SSH Server
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the right end of the tab bar.
+   You can also press `Cmd + T` to achieve the same result.
+2. Look for the SSH server you want to connect to and click the Connect button to the right.
+3. Select or enter the SSH user you wish to log in as and press `Enter`.
+4. A new tab will open with a shell session on the chosen server.
+
+Alternatively, you can type `tsh ssh` into the command bar and the autocompletion will help you
+connect to a server.
+
+## Connecting to a Database
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the end of the tab bar. You
+   can also press `Cmd + T` to achieve the same result.
+2. Select the Databases section.
+3. Look for the database server you wish to connect to and click the Connect button to the right.
+4. Select or enter the database user you with to use and press `Enter`.
+5. A new tab will open with a new connection established between your device and the database
+   server.
+
+This connection will remain active until you click the Close Connection button or close Teleport
+Connect. The port number will persist between app restarts – you can set up your favorite client
+without worrying about the port suddenly changing.
+
+### With a GUI Client
+
+To connect with a GUI client, follow the instructions in the database connection tab under the
+Connect with GUI section.
+
+### With a CLI Client
+
+The database connection tab shows the command that can be used to connect to the database. You can
+modify the database name of the connection and then click the Run button to open a new
+terminal tab with that command executed.
+
+## Connecting to Multiple Clusters
+
+After you add your first cluster, you can keep adding more by opening the profile selector in the
+top right and clicking at the *+ Add another cluster* button.
+
+At the moment Teleport Connect supports only one user per cluster. To log in as a different user,
+log out of the cluster first.
+
+## Restarting and Reconnecting
+
+Before closing, Teleport Connect will remember the tabs that you had open at the end of the session.
+Next time you open the app, Connect will ask you if you want to reopen those tabs. If you agree,
+Connect will restore connections to all resources that were active before you closed the app.
+
+At the moment, terminal tabs are restored only as far as the working directory is concerned. Teleport
+Connect doesn't attempt to re-execute commands that were in progress when the app was closed.
+
+## Troubleshooting
+
+In an attempt to resolve issues with a cluster you can try logging out of it, closing the app and
```

Comment:
What kinds of issues would this be?

Reply:
I was thinking of two types of issues:

1. A bug which results in the UI getting into a state which renders the app unusable, except for the top bar. For example, https://github.com/gravitational/webapps.e/issues/277.
2. An external change to the configuration of the cluster which renders the current profile unusable with the cluster.
   * It's hard for me to think of an example off the top of my head. But I've ran into issues in development where changing something about my Teleport cluster would render the profile stored by tsh & Connect invalid with no possibility of logging in again.
   * Usually that would be a result of doing something related to CA certs maybe, something that would be rather unlikely to happen in a live cluster but still possible.

I can try to do a general description of those issues here.